protobuf/transform.go

```diff
@@ -167,6 +182,14 @@ func (t *Transformer) transformStruct(pkg *Package, s *scanner.Struct) *Message
 	return msg
 }
 
+func defaultOptionsForScannedMessage(s *scanner.Struct) Options {
+	opts := make(Options)
+
+	opts["(gogoproto.drop_type_declaration)"] = NewLiteralValue("true")
+
+	return opts
```

Comment:
About abstracting, the problem is this:

Even if a plugin does the _same_, for a broad definition of _same_, between messages and enums, it seems to require or, at the very least, conventionally requires different name in protobuf. Currently we only include drop_type related options, but this is a great point to pluginize so people can add their default options there.